docs_samples/02_python_app_insights_dependencies.md

```diff
@@ -0,0 +1,159 @@
+# Capture Dependencies
+
+You can utilize opencensus to send dependencies like database calls and other items to Application Insights.  This is an example of how to add the various settings to a Django application to do exactly that.  You will run the samples both locally and deploy the Django web application to the Azure App Service resource you deployed as part of the setup script.
```

Comment:
Move this section down to right under the `## Configuring Django applications` header and maybe add the same kind of description here but for creating a function app.

Reply:
Got it!